mne_bids/sidecar_updates.py

```diff
@@ -0,0 +1,75 @@
+"""Update BIDS directory structures and sidecar files meta data."""
+# Authors: Adam Li <adam2392@gmail.com>
+#          Austin Hurst <mynameisaustinhurst@gmail.com>
+# License: BSD (3-clause)
+
+import json
+from collections import OrderedDict
+
+from mne_bids.utils import _write_json
+
+
+# TODO: add support for tsv files
+def update_sidecars(bids_path, sidecar_template_fpath, verbose=True):
```

Comment:
I think it's simpler to provide a `dict` ... it's less of a hassle and everything is documented in your script -- hence more reproducible

Reply:
Could be both, right? path-like or dict

Reply:
Included both.